examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -231,6 +225,16 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
               }
             });
 
+            function moveSliderHandlerToFront(){
+              if (!$scope.playButtonPaused) {
+                document.getElementById("play-button").click();
+              }
+              onPlay = false;
```

Comment:
I found `onPlay` is only used at one place. I think it can be replaced by `$scope.playButtonPaused`.